README.md

```diff
@@ -102,7 +102,17 @@ f.Close()
 
 ## S3
 
-Not yet implemented!  Watch this space.
+S3 is the default implmentation for AWS S3. This uses [aws-sdk-go/aws/session.NewSession](http://docs.aws.amazon.com/sdk-for-go/api/aws/session/#NewSession) for authentication.
```

Comment:
implmentation -> implementation :-P

Reply:
I swear I am cursed when spelling that word, will fix it up haha